tools/run_insertion_deletion.py

```diff
@@ -0,0 +1,14 @@
+from argparse import ArgumentParser, Namespace
```

Comment:
Please remove this file. It does not make sense only to parse a `work_dir` and pass it to the function. There are many other arguments like config and options to override the config. I will take care of the high-level API